vkubelet/apiserver.go

```diff
@@ -26,6 +27,21 @@ func ApiserverStart(provider Provider) {
 	p = provider
 	certFilePath := os.Getenv("APISERVER_CERT_LOCATION")
 	keyFilePath := os.Getenv("APISERVER_KEY_LOCATION")
+
+	var err error
+	
+	log.Println(certFilePath)
+	log.Println(keyFilePath)
+	
+	if certFilePath == "" || keyFilePath == "" {
```

Comment:
Should this be "&&"?

Reply:
Ahhhh yeah. Forgot to swap this when we switched to not overwriting bad combos.